akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowFlatMapPrefixSpec.scala

```diff
@@ -261,23 +261,23 @@ class FlowFlatMapPrefixSpec extends StreamSpec {
         .toMat(Sink.seq)(Keep.both)
         .run()
 
-      notUsedF.value should be(empty)
-      suffixF.value should be(empty)
-      srcWatchTermF.value should be(empty)
```

Comment:
No guarantee that notUsed (the inner flow matval) hasn't complete yet here. Asserting on termination watch future completed is covered by rest of test so skipped that.

Reply:
I was first confused about that statement, but, of course, it's because `src10()` will instantly provide elements and trigger materialization of the inner flow.